webapp/src/main/java/gov/nist/healthcare/ttt/webapp/common/controller/SVAPValidatorCtrl.java

```diff
@@ -0,0 +1,116 @@
+package gov.nist.healthcare.ttt.webapp.common.controller;

```

Comment:
This is not something that is actiionble now, but, for the future, we really need to think about why we need a backend at all for c-cda validator calls. We should be calling the SITE endpoint directly from the UI if possible. As is, it complicates and obfuscates. I'm sure there is some reason I'm not aware of, but I feel like we can work through it, or if needed, pass the backend work off to the site service.

Reply:
I think this to to pass the different variable based on the different options